Yoast/javascript
# Fix snippet preview elements order.

## Summary

This PR can be summarized in the following changelog entry:

* [search-metadata-previews] Fixes a bug where the Snippet Preview elements had misplaced visually hidden text.

## Relevant technical choices:

*

## Test instructions
- start the standalone app and go to the snippet preview tab
- since the order of the snippet elements changes between mobile and desktop, check both the mobile and desktop preview
- verify the following visually hidden text are in the correct order
- `Url preview:` before the URL
- `SEO title preview:` before the title
- `Meta description preview:` before the description

## Impact check
* This PR affects the following parts of the plugin, which may require extra testing:
  * Metabox

## UI changes

## Quality assurance


Fixes https://github.com/Yoast/bugreports/issues/563
